File: prog14.java
import java.util.*;
import java.util.Scanner;
public class prog14{
       public static void main(String args[]){
           Scanner scn=new Scanner(System.in);
           System.out.println("Enter a number");
           int n=scn.nextInt();
           int arr[]=new int[n];
           for(int i=0;i<arr.length;i++){
               arr[i]=scn.nextInt();
           }

           for(int pass=1;pass<n;pass++){
               for(int i=0;i<n-pass;i++){
                   if(arr[i]>arr[i+1]){
                       swap(arr,i,i+1);
                   }
               }
           }
           System.out.println("After Sorting:");
           for(int i=0;i<n;i++){
               System.out.print(arr[i]+" ");
           }
       }
       public static void swap(int arr[],int i,int j){
           int t=arr[i];
           arr[i]=arr[j];
           arr[j]=t;
       }
}

File: prog14.c

#include <stdio.h>
  
void swap(int* xp, int* yp)
{
    int temp = *xp;
    *xp = *yp;
    *yp = temp;
}
  

void bubbleSort(int arr[], int n)
{
    int i, j;
    for (i = 0; i < n - 1; i++)
  

        for (j = 0; j < n - i - 1; j++)
            if (arr[j] > arr[j + 1])
                swap(&arr[j], &arr[j + 1]);
}
  

int main()
{
    int s;
    printf("Enter size of array :");
    scanf("%d",&s);
    int arr[s];
    printf("Enter values in array\n");
    for(int i=0;i<s;i++){
        scanf("%d",&arr[i]);
    }
    bubbleSort(arr, s);
    printf("Sorted array: \n");
    
    for (int i = 0; i < s; i++)
        printf("%d ", arr[i]);
    printf("\n");
    return 0;
}